### PR TITLE
REFACTOR: improve import_nastran method

### PR DESCRIPTION
**Pull request by maxcapodi78**

### CODE REVIEW

**Review by Samuelopez-ansys — APPROVED** [review]

File: pyaedt/modeler/modeler3d.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 import os.path
 import warnings
 
+from line_profiler_pycharm import profile
+
```
Comment (SMoraisAnsys) [inline pyaedt/modeler/modeler3d.py]:
```suggestion
```

File: pyaedt/modeler/modeler3d.py [hunk 2]
```diff
@@ -877,7 +879,8 @@ def objects_in_bounding_box(self, bounding_box, check_solids=True, check_lines=T
         return objects
 
     @pyaedt_function_handler()
-    def import_nastran(self, file_path, import_lines=True, lines_thickness=0, **kwargs):
+    @profile
```
Comment (SMoraisAnsys) [inline pyaedt/modeler/modeler3d.py]:
```suggestion
```

**Review by SMoraisAnsys — COMMENTED** [review]
We this LOCs used to debug only or are they supposed to be kept ?

**Review by Samuelopez-ansys — APPROVED** [review]
LGTM!

**Pull request merged**